draft-irtf-cfrg-hpke.md

```diff
@@ -271,52 +273,108 @@ A set of algorithm identifiers for concrete instantiations of these
 primitives is provided in {{ciphersuites}}.  Algorithm identifier
 values are two bytes long.
 
+The following two functions are defined for a KDF to facilitate domain
+separation of calls as well as context binding:
+
+~~~
+def LabeledExtract(salt, label, IKM):
```

Comment:
You use `0` for the salt value several times below.  It would be good to clarify whether that means (a) an all-zero octet string (and if so, what length), or (b) the empty octet string.

Reply:
In commit https://github.com/cfrg/draft-irtf-cfrg-hpke/pull/50/commits/e7313ea2845c72a543dcc32f1eb2be99acb2d9d9, I chose to make explicit what HKDF-Extract would do by default, by using `zero(Nh)`.